docs/rule/no-obscure-hbs.md

```diff
@@ -0,0 +1,39 @@
+# no-obscure-hbs
```

Comment:
We should name this something specific to this issue like `no-obscure-array-access`.

Reply:
renamed